app/helpers/users_helper.rb

```diff
@@ -1,4 +1,10 @@
 # frozen_string_literal: true
 
 module UsersHelper
+  def flash_class(level)
```

Comment:
Maybe `flash_class` in here and problem_reports_helper should be moved to ApplicationHelper instead?

Reply:
o.o this exists in application_helper, users_helper, problem_reports_helper